src/xtopcom/xvledger/src/xvblock.cpp

```diff
@@ -1038,6 +1045,10 @@ namespace top
         
         const std::string  xvqcert_t::get_hash_to_sign() const //signatrure' hash = hash(m_target_obj_hash + m_consensus + m_cryptos..)
         {
+            if (0 == m_modified_count && m_sign_hash.empty() == false)  {
```

Comment:
for Boolean function, !m_sign_hash.empty() better?
no race condition?

Reply:
ok